BUILD.bazel

```diff
@@ -1027,6 +1027,152 @@ _RPI_OVERLAY_OUTS = [
+    "sound/soc/codecs/snd-soc-adau1977.ko",
+    "sound/soc/bcm/snd-soc-audioinjector-pi-soundcard.ko",
+    "sound/soc/bcm/snd-soc-rpi-simple-soundcard.ko",
+    "sound/soc/bcm/snd-soc-bcm2835-i2s.ko",
+    "sound/soc/bcm/snd-soc-justboom-both.ko",
+    "sound/soc/bcm/snd-soc-hifiberry-dacplushd.ko",
+    "sound/soc/bcm/snd-soc-63xx.ko",
+    "sound/soc/codecs/snd-soc-pcm1794a.ko",
+    "drivers/gpu/drm/bridge/simple-bridge.ko",
+    "drivers/clk/clk-hifiberry-dachd.ko",
+    "sound/soc/bcm/snd-soc-i-sabre-q2m.ko",
+    "sound/soc/codecs/snd-soc-tlv320aic32x4.ko",
+    "sound/soc/bcm/snd-soc-audiosense-pi.ko",
+    "sound/soc/bcm/snd-soc-justboom-dac.ko",
+    "drivers/media/usb/gspca/gspca_main.ko",
+    "sound/soc/codecs/snd-soc-sgtl5000.ko",
+    "sound/soc/codecs/snd-soc-wm5102.ko",
+    "sound/soc/bcm/snd-soc-allo-piano-dac.ko",
+    "sound/soc/codecs/snd-soc-tlv320aic32x4-i2c.ko",
+    "sound/soc/codecs/snd-soc-da7213.ko",
+    "sound/soc/bcm/snd-soc-hifiberry-dacplus.ko",
+    "sound/soc/bcm/snd-soc-allo-katana-codec.ko",
+    "drivers/gpu/drm/bridge/display-connector.ko",
+    "sound/soc/codecs/snd-soc-adau1977-i2c.ko",
+    "sound/soc/codecs/snd-soc-wm-adsp.ko",
+    "sound/soc/codecs/snd-soc-tas571x.ko",
+    "sound/drivers/snd-dummy.ko",
+    "sound/soc/bcm/snd-soc-hifiberry-dacplusdsp.ko",
+    "drivers/gpu/drm/panel/panel-ilitek-ili9806e.ko",
+    "sound/soc/codecs/snd-soc-arizona.ko",
+    "sound/soc/bcm/snd-soc-digidac1-soundcard.ko",
+    "sound/soc/bcm/snd-soc-chipdip-dac.ko",
+    "drivers/gpu/drm/drm_mipi_dbi.ko",
+    "sound/soc/bcm/snd-soc-googlevoicehat-codec.ko",
+    "sound/soc/codecs/snd-soc-wm8731.ko",
+    "sound/soc/bcm/snd-soc-pifi-40.ko",
+    "sound/soc/codecs/snd-soc-pcm186x-i2c.ko",
+    "drivers/rtc/rtc-ds1302.ko",
+    "drivers/rtc/rtc-ds1390.ko",
+    "drivers/leds/leds-pwm.ko",
+    "arch/arm64/crypto/poly1305-neon.ko",
+    "sound/soc/codecs/snd-soc-dmic.ko",
+    "drivers/rtc/rtc-ds1672.ko",
+    "drivers/rtc/rtc-ds1343.ko",
+    "sound/soc/codecs/snd-soc-pcm5102a.ko",
+    "sound/soc/generic/snd-soc-simple-card-utils.ko",
+    "drivers/rtc/rtc-ds1347.ko",
+    "drivers/rtc/rtc-ds1286.ko",
+    "lib/crypto/libchacha20poly1305.ko",
+    "fs/efivarfs/efivarfs.ko",
+    "sound/soc/codecs/snd-soc-tpa6130a2.ko",
+    "drivers/rtc/rtc-ds1305.ko",
+    "drivers/rtc/rtc-max6916.ko",
+    "sound/soc/codecs/snd-soc-i-sabre-codec.ko",
+    "sound/soc/codecs/snd-soc-pcm179x-i2c.ko",
+    "drivers/rtc/rtc-ds1511.ko",
+    "drivers/usb/class/cdc-acm.ko",
+    "arch/arm64/crypto/chacha-neon.ko",
+    "lib/crypto/libcurve25519.ko",
+    "drivers/rtc/rtc-max6900.ko",
+    "drivers/pwm/pwm-pca9685.ko",
+    "drivers/rtc/rtc-ds1374.ko",
+    "sound/soc/codecs/snd-soc-pcm186x.ko",
+    "sound/soc/generic/snd-soc-simple-card.ko",
+    "sound/soc/codecs/snd-soc-pcm512x-i2c.ko",
+    "sound/soc/codecs/snd-soc-pcm512x.ko",
+    "sound/soc/codecs/snd-soc-pcm179x-codec.ko",
+    "lib/crypto/libcurve25519-generic.ko",
+]
```

Comment:
@gajavrbinc how did you create this list?

Reply:
@JanKehren When compiling using the config file, bazel finds the modules (=m), returns an error and writes these paths based on which modules were specified in the config file. After adding these correct paths, bazel compiled succesfully.